src/main/scala/ir/Expr.scala

```diff
@@ -354,9 +355,13 @@ sealed trait Variable extends Expr {
 
   def size: Int = irType match {
     case b: BitVecType => b.size
-    case _ => throw new Exception("tried to get size of non-bitvector")
+    case _             => throw new Exception("tried to get size of non-bitvector")
   }
 
+  val isRegister: Boolean = name.startsWith("R") || name.startsWith("V") && name != "VF"
+
+  // `rName` : e.g. "R20"
+  def isRegister(rName: String): Boolean = isRegister && rName.equals(name)
```

Comment:
This is redundant now that we have a separate Register case class, and that should be used instead